File: ed/c/examples/whatever/hw.c
#include <stdio.h>

static int z = 3;

void hello(int y)
{
    int x = 1;
    printf("hellow world: %d, %d, %d \n", x, y, z); // hellow world: 1, 2, 3
}

int main()
{
    hello(2);
    return 0;
}

File: ed/l/python/examples/whatever/hw.py
"""
This is simple hw script.
"""


def f2():
    name = "Bond"
    print("hi {0}".format(name))
    print(f"hi {name}")


def f():
    """
    multiline string.
    """
    str = "Hello" \
          " World!"
    print(f"{str} \n")


f()
f2()
